### ISSUE TITLE
Image sets for the player

### ISSUE THREAD

**Issue by Quintus** [Secretchronicles/TSC#418]
I’m not sure why @brianvanderburg2 didn’t create `.imgset` files for the player, so here’s an issue to ask him :-). And ideally of course, add `.imgset` files for the player. The graphics in question are in the `pixmaps/alex` subdirectories.

Valete,
Quintus

**Comment by brianvanderburg2** [comment]
I've created some image sets for the player and push them in a branch called vanderburg_player_imgset.  There are still other issues, some perhaps quite major:

* Loading the images with Add_Image_Set.  This is quite easy.  I've already changed the code, just haven't committed or pushed it yet. However, from looking at the original code that adds the images, it looks like the image positions for each player status (small/big/fire/etc) are the same for each.  This can't be guaranteed with image sets because an image set may contain any number of images.  I don't really see why this needs to be guaranteed, when changing from one animation to another we would just start that new animation sequence from the start.
* The timings are most definitely wrong and will need to be tweaked once the code is adjusted to use them.
* Some areas in the code may adjust the animation speed (such as walking on ice).  If this is the case, it should do this by using Set_Animation_Speed.  Another idea is to modify cImage_Set to support an animation_speed property to Update_Animation to temporarily override the current animation speed.
```c++
cImage_Set::Update_Animation(float animation_speed = -1.0)
{
    if(animation_speed < 0.0) { animation_speed = m_animation_speed; }
    ....
}
```
* No code in the player uses Update_Animation.  It seems that the current image is calculated manually for everything in various Update_... methods and Get_Image (which happens to hide cImage_Set::Get_Image).  This will need to be adjusted to use Set_Image_Set(name) and Set_Animation(on_or_off), and to call Update_Animation().  As the player moves to different ground types, Set_Animation_Speed can be called to speed up or slow down the animation, with 1.0 being the baseline speed.

**Comment by Quintus** [comment]
Okay, thank you for the info @brianvanderburg2 :-). In my ongoing effort to port stuff to SFML the code loading the graphics in `cLevel_Player` has to be revised anyway, so I take the opportunity and adapt it to use the same interface as every other sprite does also. Unifying this will keep future effort low anyway.

Vale,
Quintus

**Comment by brianvanderburg2** [comment]
Okay.  I also had some other ideas about image sets in #425

**Comment by Quintus** [comment]
This has long been fixed (commit 4cc405df30c6db2bff4ce4ede165731ca05bf830).
